src/components/CytoscapeGraph/graphs/GraphStyles.ts

```diff
@@ -331,8 +331,10 @@ export class GraphStyles {
           break;
         }
         case EdgeLabelMode.RESPONSE_TIME_95TH_PERCENTILE: {
-          const responseTime = edgeData.responseTime > 0 ? edgeData.responseTime : 0;
-          if (responseTime > 0) {
+          // hack to fix responseTime is sometimes a string during runtime even though its type is number
+          const responseTimeNumber = parseInt(String(edgeData.responseTime));
+          const responseTime = responseTimeNumber > 0 ? responseTimeNumber : 0;
+          if (responseTime && responseTime > 0) {
```

Comment:
This works as a fix and we should probably merge in a fix just for safety in the immediate-term, but I'm concerned that we are somehow getting compromised or unexpected response time values and I'd like to know the root cause.  Can we add something that dumps out the string values when we hit this? Maybe like:

```
let responseTime = edgeData.responseTime;
if ( !Number.isInteger(responseTime) ) {
  console.log(`Unexpected string responseTime=|${responseTime}|`);
  responseTime = parseInt(responseTime);
}
```

I mean it's weird!  We could even dump out the entire graph json because I wonder what is getting returned.

Reply:
@jshaughn sure, good idea

Reply:
That ended up spewwing more in the logs than intended:

![Kiali_Console](https://user-images.githubusercontent.com/1312165/77586166-f016ba00-6ea2-11ea-9030-ace042cc5339.png)

So I changed the boolean expression to ignore the NaN.